### PR TITLE
test_setup: fix incorrect format

### PR DESCRIPTION
**Pull request by dzhengfy**
Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

File: virttest/test_setup/__init__.py [hunk 1]
```diff
@@ -533,8 +533,9 @@ def set_node_num_huge_pages(self, num, node, pagesize, ignore_error=False):
         # If node has some used hugepage, result will be larger than expected.
         if obj.sys_fs_value < int(num):
             msg = "%s (expecting %s) hugepages is set on node %s, "
-            "please check if the node "
-            "has enough memory" % (obj.sys_fs_value, num, node)
+            msg += "please check if the node has enough memory"
```
Comment (chloerh) [inline virttest/test_setup/__init__.py:536]: How about put '\' at the end?

Comment (dzhengfy) [reply virttest/test_setup/__init__.py]: Agree. Updated.

**Review by chloerh — APPROVED** [review]
There're trailing spaces that need to be removed

**Comment by dzhengfy** [comment]
> There're trailing spaces that need to be removed

Done

**Pull request merged**